docs/releases/legacy-pre-0.1-history.md

```
@@ -0,0 +1,587 @@
+### Bug Fixes
+- business/team workspace selection: detect multiple workspace account IDs from oauth tokens and prompt for the correct one.
+- prevent refresh/hydration from overwriting selected workspace ids (org/manual choices remain stable).
+- persist workspace labels and sources for clearer account listings.
+
+### Changelog
+
+Full Changelog: https://github.com/ndycode/codex-multi-auth/compare/v4.9.5...v4.9.7
+
+## [4.9.5] - 2026-01-28
+
+### Improvements
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)